doc/api/fs.md

```diff
@@ -3528,6 +3528,12 @@ On Linux, positional writes don't work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+On Windows, if the file descriptor is a TTY, e.g. a terminal with file
+descriptor `1` / standard output, a string containing utf8 encoded output will
```

Comment:
The "e.g. ... output," is confusing, because a terminal with fd not equal to 1 would still be a tty, and I don't know what the  "/ " means. I suggest removing that part, then adding a second sentence afterwards: "Usually, file descriptors 0, 1, and 2 (also known as `stdin`, `stdout`, and `stderr`) are connected to a terminal. For any file descriptor, whether it is a terminal can be checked using  [`isatty()`](https://nodejs.org/api/all.html#tty_tty_isatty_fd)".

Reply:
Nit: it seems we usually use uppercased UTF8 or UTF-8 outside of code snippets.

Reply:
I'd skip the referenced to `TTY` at all since IIUC the crux is `conhost.exe` limitations WRT to stdio. That is, not the type of the fd, but where it is connected to (i.e. a TTY connected to a pipe or socket will work as expected).